src/helpers/keycard.ts

```diff
@@ -0,0 +1,21 @@
+import { Keycard } from '@snapshot-labs/keycard';
+import { sendError } from './utils';
+
+const keycard = new Keycard({
+  app: 'snapshot-hub',
+  secret: process.env.KEYCARD_SECRET || '',
+  URL: process.env.KEYCARD_URL || 'https://keycard.snapshot.org'
+});
+
+const verifyKeyCard = async (req, res, next) => {
+  const key = req.headers['x-api-key'] || req.query.apiKey;
+  if (key && keycard.configured) {
```

Comment:
I don't understand what is "key is not settled" 🙈 can explain?

This line is used to check if a request has an API key or not and check if the keycard is configured or not. so that it won't block the current flow/local env if we don't have `KEYCARD_SECRET` in .env